src/NServiceBus.Core/Recoverability/FirstLevelRetries/ProcessingFailureInfo.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace NServiceBus
+{
+    using System;
+
+    class ProcessingFailureInfo
```

Comment:
My take on this would also be a no. If there's a chance that TM might be taken out of the Core, then that might be a reason to keep them separate.